sandy/decay.py

```diff
@@ -114,6 +114,40 @@ def get_halflives(self):
         series.index.name = "ZAM"
         return series
 
+    def get_decayconstant(self, with_uncertainty=True):
+        """
+        Extract decay constant and its uncertainty into dataframe.
+
+        Parameters
+        ----------
+        with_uncertainty : `bool`, optional, default is 'True'
+
+        Returns
+        -------
+        `pandas.DataFrame`
+             decay constant dataframe
+
+        Examples
+        --------
+        >>> endf6 = sandy.get_endf6_file("jeff_33", "decay", 922350)
+        >>> rdd = sandy.DecayData.from_endf6(endf6)
+        >>> DC = sandy.DecayData.get_decayconstant(rdd)
+        >>> print(DC)
+                   DLAMBDA      LAMBDA
+           922350 2.21715e-20 3.12085e-17
+        """
+
+        decay_constant = {zam: {
+             "LAMBDA": dic['decay_constant'],
+             "DLAMBDA": dic['decay_constant_uncertainty'],
+             } for zam, dic in self.data.items()}
+
+        df = pd.DataFrame(decay_constant)

```

Comment:
I would do
```
df = pd.DataFrame(decay_constant).T
if with_uncertainty:
            return df
else:
            return df.LAMBDA
```
I am not sure `iat` is the best for this case

Reply:
But if you do this you need to say in the docstring that you get a series in output if with_uncertainty is false

Reply:
Let me know if what I wrote is the correct way to explain the two output options